components/MenteesList/MenteeCard.js

```diff
@@ -32,68 +21,37 @@ const MenteeCard = ({ mentorship }) => {
       <Stack
         direction={{ base: "column", lg: "row" }}
         spacing={{ base: 4, lg: 20 }}
+        width='100%'
       >
-        <Stack
-          pl={4}
-          direction={"column"}
-          textAlign={"center"}
-          justifyContent={"center"}
-        >
-          <Avatar
-            size="xl"
-            src={mentee.picture + "?tr=w-100,h-100,fo-auto"}
-            name={mentee.name}
-            alignSelf={"center"}
-          />
-          <Heading size="sm">{mentee.name}</Heading>
-          <Stack
-            direction={"row"}
-            textAlign={"center"}
-            justifyContent={"center"}
-          >
-            {mentee.twitter && (
-              <NextLink href={mentee.twitter} passHref>
-                <Link target="_blank">
-                  <Icon color="#1DA1F2" as={FaTwitterSquare} />
-                </Link>
-              </NextLink>
-            )}
-            {mentee.linkedin && (
-              <NextLink href={mentee.linkedin} passHref>
-                <Link target="_blank">
-                  <Icon color="#2867B2" as={FaLinkedin} />
-                </Link>
-              </NextLink>
-            )}
-          </Stack>
-        </Stack>
-
-        <Stack
-          direction={"column"}
-          textAlign={"left"}
-          justifyContent={"center"}
-        >
-          <ReactMarkdown components={ChakraUIRenderer(markdownTheme)} skipHtml>
-            {mentee.biography}
-          </ReactMarkdown>
-        </Stack>
-        <Stack
-          direction={"column"}
-          textAlign={"left"}
-          justifyContent={"center"}
-        >
-          <ReactMarkdown components={ChakraUIRenderer(markdownTheme)} skipHtml>
-            {mentorship.message}
-          </ReactMarkdown>
-        </Stack>
-
-
+        <PersonInfo person={mentee} />
+        <MentorshipInfo mentorship={{ message, mentorshipDate, status, longterm }} />
         <Stack
           direction={"column"}
           textAlign={"right"}
           justifyContent={"center"}
         >
-          <Button colorScheme={"greenButton"}>Details</Button>
+          <Menu>
+            <MenuButton
+              as={Button}
+              rightIcon={<ChevronDownIcon />}
+            >
+              Close
+            </MenuButton>
+            <MenuList>
+              <MenuItem 
+                onClick={() => handleCloseMentorship(mentorship, RelationshipStatusEnum.ClosedCompleted)}
+              >Completed</MenuItem>
```

Comment:
The `handleCloseMentorship` function is not defined yet, will this be created in another PR?

Reply:
Ah we just need to pass it down as a prop from MenteesList/index.js

Reply:
done